net6_bread_be/appsettings.Development.json

```diff
@@ -4,6 +4,9 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "BreadDbConnection": "Host=localhost;Port=5433;Username=postgres;Password=password123;Database=BreadDB"
```

Comment:
Is your local db on pgAdmin called "BreadDB" or "net6_bread_be"?

Reply:
I fixed this, it wasn't pushed up but it is now